src/sentry/static/sentry/app/views/releases/list/index.tsx

```diff
@@ -326,6 +328,18 @@ class ReleasesList extends AsyncView<Props, State> {
             <PageHeader>
               <PageHeading>{t('Releases')}</PageHeading>
             </PageHeader>
+
+            <Feature features={['releases-top-charts']} organization={organization}>
+              {/* We are only displaying charts if single project is selected */}
+              {selection.projects.length === 1 && !selection.projects.includes(-1) && (
```

Comment:
You can import `ALL_ACCESS_PROJECTS` from `constants/globalSelectionHeader` instead of the -1 bare number.

Reply:
Good idea 👍